integration/integration_test.go

```diff
@@ -103,33 +98,35 @@ func (s *BaseSuite) createComposeProject(c *check.C, name string) {
 	c.Assert(err, checker.IsNil)
 	s.composeProject = composeProject
 
-	s.listenChan = make(chan project.ProjectEvent)
+	s.started = make(chan bool)
```

Comment:
We should find a defer close(s.started) somewhere (and also for other channels) :)